src/tactic/apply_fun.lean

```diff
@@ -75,6 +131,7 @@ end
 meta def apply_fun (q : parse texpr) (locs : parse location) (lem : parse (tk "using" *> texpr)?)
   : tactic unit :=
 match locs with
+| (loc.ns [none]) := apply_fun_to_goal q lem
 | (loc.ns l) := l.mmap' $ option.mmap $ λ h, get_local h >>= apply_fun_to_hyp q lem
 | wildcard   := local_context >>= list.mmap' (apply_fun_to_hyp q lem)
```

Comment:
This does not allow to apply a function to the goal and some local assumptions at the same time. This is probably a very rare use case, but it will be very confusing if encountered.